### ISSUE TITLE
Feature Request: Limb Paralysis

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#20353]
So as far as I'm aware, the only way to become a cripple who needs a wheelchair is to lose your legs, like straight up just not have legs any more.
This kinda sucks because honestly I like wearing shoes. 
So what I'd like to request is some kind of limb paralysis disability that's available at round start. So you know, you can pick if you have no use of your legs, or arms, or maybe individual arms, at roundstart. So you can be a cripple, without having to be an armless weirdo who can't wear gloves or shoes or powergame that they can't be cuffed because they don't have two hands.

Additional thought: After chugging painkillers with two broken feet and a broken leg, you can still walk. This is silly nonsense, perhaps make this paralysis thing occur too if the bone damage is too great, so you can't just walk around huffing oxy with all broken limbs like you didn't just get ran train by 6 toolboxes.

In short, give me no use of limbs but still have limbs so I can wheel myself around in a wheelchair with my legs but people can't just unbuckle me and shake me up and go "you can walk retard". Ruining my fun.

**Comment by Killette2** [comment]
Doesn't shutdown syndrome do this?

**Comment by strawberrytan** [comment]
just like overdose on rezadone until your legs are as strong as a glass jackhammer

**Comment by ghost** [comment]
Shutdown syndrome might do that, I don't know I haven't looked at the code for it but I might later.
